content/attacks/posts/2022-10-11-Mango-Markets.md

```diff
@@ -0,0 +1,38 @@
+---
+date: 2022-10-11
+target-entities: 
+- Mango Markets
+entity-types: 
+- DeFi
+- Exchange
+attack-types: 
+- Price Oracle Manipulation
+title: "Mango Markets Exploited for $116 Million"
+---
+
+## Summary
+
+On October 11th, 2022, Mango Markets, a decentralized exchange on Solana, was exploited. The hacker manipulated the price oracle for the protocol's MNGO token. The hacker first took out a long MNGO position on Mango. Then the hacker artificially raised the price of the MNGO token by taking advantage of low liquidity on secondary markets. The hacker then used the temporary high price of MNGO to take out loans of USDC, various other stable coins, and SOL against unrealized profit on the long MNGO position. 
```

Comment:
Please, rewrite the summary, with less `the hacker` word. Rephrase or combine sentences, and use synonyms as `attacker` etc.